iroha/src/sumeragi.rs

```diff
@@ -17,8 +17,12 @@ use rand::{rngs::StdRng, seq::SliceRandom, SeedableRng};
 use std::{
     fmt::{self, Debug, Formatter},
     sync::Arc,
+    time::{Duration, SystemTime},
 };
 
+/// Specifies the amount of time in which the peer expects commit message from proxy after voting on the block.
+pub const COMMIT_TIME: Duration = Duration::from_millis(1000);
```

Comment:
Misleading naming, it's not the time for commit, it's time to wait for commit message. Maybe something like "COMMIT_MESSAGE_TIMEOUT"?

Reply:
It's also from the `whitepaper`. There we have `commit time` and `block time` so I try to reference them.

`after a node signs off on a block and forwards it to the proxy tail, they expect a commit message within a reasonable amount of time (the commit time);`

Reply:
I really need to read it one more time))))